src/hermes/commands/workflow.py

```diff
@@ -224,11 +224,39 @@ def deposit(click_ctx: click.Context, auth_token, file):
 
 
 @click.group(invoke_without_command=True)
-def postprocess():
+@click.pass_context
+def postprocess(click_ctx: click.Context):
     """
     Postprocess metadata after deposition
     """
-    click.echo("Post-processing")
+    _log = logging.getLogger('cli.postprocess')
+
+    audit_log = logging.getLogger('audit')
+    audit_log.info("# Post-processing")
+
+    ctx = CodeMetaContext()
+
+    if not (ctx.hermes_dir / "deposit").exists():
+        _log.error("You must run the deposit command before post-process")
+        click_ctx.exit(1)
+
+    # Get all harvesters
+    postprocess_config = config.get("post")
+    postprocess_names = postprocess_config.get('process', [])
```

Comment:
How about calling the list `plugins`, because it lists plugins which are to be invoked during the post-processing step.

To be consistent here, one could also call the `form` list in the `harvest` section `plugins`, because ... the harvest step.

Reply:
I hereby declare we do `[postprocess]` (as we already had explained in the details at #110) and name the option `execute`, because we want to declare which postprocessors shall be executed.

Sorry for being pushy here, but we need this done now for the presentation. We can change later.